README.md

```diff
@@ -84,6 +84,7 @@ Agenda's are posted to https://github.com/ethereum/pm/issues. Anyone is welcome
 - The topic involves the Ethereum protocol at a low-level. This means Dapps and ERCs are generally not allowed as topics.
 - The topic should not be philosophical. The core developer meetings are not meant to decide philosophical contentious issues that should be decided by the community. There are exceptions to this, but generally these topics distract from more productive technical discussion.
 
-### Who Manages the Meetings
-In the beginning (2015-fall 2016) George Hallam facilitated the meetings. Since the fall of 2016 Hudson Jameson has facilitated and recorded the meetings. In early 2018 Lane Rettig joined to help facilitate meetings, handle recordings, and publish notes from the meetings.
-The meetings are independent of any organization. However, Hudson Jameson is a contractor for the Ethereum Foundation and the Ethereum Foundation pays for the videoconference software used in the meetings. Livestreaming instruction for the meetings can be found [here](https://docs.google.com/document/d/1dF_Drs56ErV6wJgFmbzpdN-cINshKCsB61D92NS2JDg/edit).
+### Who Manages the Meetings?
+In the beginning (2015-fall 2016) George Hallam facilitated the meetings. Since the fall of 2016 [Hudson Jameson](https://github.com/souptacular) has facilitated and recorded the meetings. In early 2018 [Lane Rettig](https://github.com/lrettig) began helping facilitate meetings, handle recordings, and publish notes from the meetings. In 2019, [Tim Beiko](https://github.com/timbeiko) and the Ethereum Cat Herders began to help with these tasks as well.
```

Comment:
Perhaps we should link the ECH gitter or PM repo so that people who aren't familiar can know where to go.